src/main/battlecode/world/GameWorld.java

```diff
@@ -129,6 +130,8 @@ public GameWorld(GameMap gm, RobotControlProvider cp,
                     Optional.empty()
             );
         }
+        
+        rand = new Random();

```

Comment:
We need to keep GameWorld deterministic (/ consistently producing the same results on the same map), so let's seed this Random with the game's default map seed.